Configuration/Applications/python/Options.py

```diff
@@ -428,6 +428,12 @@
                             type=str,
                             dest="heap_profile")
 
+expertSettings.add_argument("--maxmem_profile",
+                            help="add the PerfTools/MaxMemoryPreload Service",
```

Comment:
It isn't really Service, so I'd use a different word to avoid confusion. How about
```suggestion
                            help="add the PerfTools/MaxMemoryPreload monitor",
```
?